### ISSUE TITLE
Godot cannot open Visual Studio code with editor settings...

### ISSUE THREAD

**Issue by hislittlecuzin** [godotengine/godot#76342]
### Godot version

4.0.2 mono

### System information

Windows 11 Lenovo duet 5i

### Issue description

Brand new windows installation. (Not important)

Visual Studio code will not open c# files AND c# files will not open in the Godot built in text editor. 

### Steps to reproduce

Download Godot. Download visual studio code. 

In Godot, click the Editor drop-down menu. 
Under the "Text Editor" expandable section, open "External."
This is the settings you should set. 
Set "use external editor" true. 
Set "Exec Path" to your "code.exe" file in your visual studio code directory. 
Exec Flags can stay default. For me, default is "{file}"

To cause bug, go to the "Dotnet" expandable menu (towards the bottom)
Click "Editor" menu. 
Set "External Editor" to "Visual studio Code."

The workaround is to set the "external editor" in the "Dotnet" section to be "Disabled."

### Minimal reproduction project

A scene with a c# script on it. Save it. 
I used a 3d scene. 
Basically trying to do hello world with 3d scene using console output.

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot/issues/76627.

How did you install Visual Studio Code? Did you use the official installer – and if so, which one? Do you get any error messages in the Output panel when trying to edit a script with an external editor configured?

**Comment by Calinou** [comment]
Closing due to lack of response. Please comment if you can still reproduce this bug on the latest Godot version.
